linux/deploy_vm/delete_unattend_install_iso.yml

```diff
@@ -1,17 +1,8 @@
-# Copyright 2021-2023 VMware, Inc.
+# Copyright 2023 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
-# Eject CDROM of unattend install ISO in guest OS
```

Comment:
Yes, because now OS will be shutdown after autoinstall completes. And this task is executed after shutdown which means VM is powered off at that time.